server/middlewares/authentications/verifyToken.js

```diff
@@ -0,0 +1,24 @@
+import dotenv from "dotenv";
+import { getJWTConfigs, decodeJWT } from "../../helpers/jwt";
+import { httpResponse, serverError } from "../../helpers/http";
+
+const tokenConfigs = getJWTConfigs({ subject: "Authentication Token" });
```

Comment:
This is hardcoded and is not scalable. Please use the `getJWTConfigs` object that already is in use. We don't need to be changing variables for each usage.